docs/content/en/docs/getting-started/execution.md

```diff
@@ -4,67 +4,128 @@ weight: 3
 description: "Execution traces with Tetragon"
 ---
 
-At the core of Tetragon is the tracking of all executions in a kubernetes cluster,
-virtual machines, and baremetal systems. This creates the foundation that allows
+At the core of Tetragon is the tracking of all executions in a Kubernetes cluster,
+virtual machines, and bare metal systems. This creates the foundation that allows
 Tetragon to attribute all system behavior back to a specific binary and its
 associated metadata (container, Pod, Node, and cluster).
 
-## Observe Tetragon Execution Events
+## Observe Tetragon execution events
 
-Tetragon exposes the execution trace over JSON logs and GRPC stream. The user
+Tetragon exposes the execution events over JSON logs and GRPC stream. The user
 can then observe all executions in the system.
 
-The following command can be used to observe exec events.
+Use the following instructions to observe execution events. These instructions
+assume you have deployed the Cilium [demo application](https://docs.cilium.io/en/stable/gettingstarted/demo/)
+in your environment.
 
-{{< tabpane lang=shell >}}
-{{< tab Kubernetes >}}
+{{< tabpane text=true >}}
+{{% tab "Kubernetes (single node)" %}}
+
+For a single node Kubernetes cluster, you can target the Tetragon DaemonSet with
+a `kubectl exec` command:
+
+```shell
 kubectl exec -ti -n kube-system ds/tetragon -c tetragon -- tetra getevents -o compact --pods xwing
-{{< /tab >}}
-{{< tab Docker >}}
-docker exec tetragon-container tetra getevents -o compact
-{{< /tab >}}
-{{< /tabpane >}}
+```
+
+This command runs `tetra getevents -o compact --pods xwing` in the single Pod
+that is a member of the Tetragon DaemonSet. Because there is only a single node
+in the cluster, it is guaranteed that the "xwing" Pod will also be running on
+the same node and that Tetragon will be able to capture and report execution
+events.
+
+{{% /tab %}}
+{{% tab "Kubernetes (multiple nodes)" %}}
+
+In a cluster with multiple nodes, you will first need to find the node where
+the workload is running. This command shows the node where the "xwing" Pod is
+running.
+
+```shell
+kubectl get pod xwing -o jsonpath='{.spec.nodeName}'
+```
+
+Next, you will need to find the Tetragon Pod on the same node. This command
+returns a list of Tetragon Pods and the name of the Kubernetes node on which
+they are running.
 
-This will print a compact form of the exec logs. For an example we do the following
-with the demo application.
+```shell
+kubectl -n kube-system get pods -o custom-columns=NAME:.metadata.name,NODE:.spec.nodeName | grep <name-of-node-with-workload>
+```
+
+Once you have the identified the matching Pod, then target it with a `kubectl
+exec` to run the `tetra getevents` command.
+
+```shell
+kubectl exec -ti -n kube-system po/<tetragon-pod-name> -c tetragon -- tetra getevents -o compact --pods xwing
+```
```

Comment:
Could you condense those three commands into two using shell variable so that it's "copy pastable"?

```shell
POD=$(kubectl get pod -l 'app.kubernetes.io/name=tetragon' -o name -n kube-system --field-selector spec.nodeName=$(kubectl get pod xwing -o=jsonpath='{.spec.nodeName}'))
// stuff
kubectl exec -ti -n kube-system $POD -c tetragon -- tetra getevents -o compact --pods xwing
```

Reply:
For the record, I'm not a fan of a single command that doesn't explain what it's doing.

I'd rather get these changes into the docs than get hung up on a minor difference of opinion, so I've added the two commands as suggested above in commit bc4204e.